src/System.Private.CoreLib/shared/System/Text/ASCIIUtility.cs

```diff
@@ -1395,17 +1336,7 @@ private static unsafe nuint NarrowUtf16ToAscii_Sse2(char* pUtf16Buffer, byte* pA
             // Turn the 8 ASCII chars we just read into 8 ASCII bytes, then copy it to the destination.
 
             Vector128<byte> asciiVector = Sse2.PackUnsignedSaturate(utf16VectorFirst, utf16VectorFirst);
-
-            if (Sse41.X64.IsSupported)
-            {
-                // Use PEXTRQ instruction if available, since it can extract from the vector directly to the destination address.
-                Unsafe.WriteUnaligned<ulong>(pAsciiBuffer, Sse41.X64.Extract(asciiVector.AsUInt64(), 0));
-            }
-            else
-            {
-                // Bounce this through a temporary register (with potential stack spillage) before writing to memory.
-                Unsafe.WriteUnaligned<ulong>(pAsciiBuffer, asciiVector.AsUInt64().GetElement(0));
-            }
+            Unsafe.WriteUnaligned<ulong>(pAsciiBuffer, asciiVector.AsUInt64().GetElement(0));
```

Comment:
What's the codegen for this here?

Reply:
Per our offline conversation I've modified this to `Sse2.StoreLow`. Will get the codegen in just a sec.

Reply:
Codegen is optimal after the switch to `Sse2.StoreLow`. Here's the codegen for the three uses within this method:

```asm
00007ff9`cc53dbdb c5f9d612        vmovq   mmword ptr [rdx],xmm2
00007ff9`cc53dc04 c5f9d65208      vmovq   mmword ptr [rdx+8],xmm2
00007ff9`cc53dc67 c5f9d61402      vmovq   mmword ptr [rdx+rax],xmm2
```